games/fizzbuzz.py

```diff
@@ -0,0 +1,15 @@
+def fizzbuzz():
+    number = int(input("Choose a number: "))
+    for i in range(number):
+        print(i)
+    if number % 3 == 0 and number % 5 == 0:
```

Comment:
Major: missing one tab of indentation from line 5 to line 12, it should print "1,2,fizz,4,buzz.." but right now it prints the whole number sequence and then either fizz or buzz only at the end 🙂

Reply:
ow so that's what you meant, I though of another variant where only the ending number was supposed to go through the fizzbuzz process, my bad